doc/src/alerts/developing_alerts.rst

```diff
@@ -179,107 +151,369 @@ Creating Flow Alerts
+
+	HTTPContacts::HTTPContacts()
+		: HostCheck(ntopng_edition_community, false /* All interfaces */,
+					false /* Don't exclude for nEdge */,
+					false /* NOT only for nEdge */){};
+
+	void HTTPContacts::periodicUpdate(Host *h, HostAlert *engaged_alert) {
+	HostAlert *alert = engaged_alert;
+	u_int8_t num_http_flows = 0;
+
+	num_http_flows = h->getNumHttpFlows();
+
+	if (num_http_flows > threshold) {
+		if (!alert)
+		alert =
+			allocAlert(this, h, CLIENT_FAIR_RISK_PERCENTAGE, num_http_flows, threshold);
+		if (alert) {
+		h->triggerAlert(alert);
+		h->resetNumHttpFlows();
+		} 
+	}
+	}
+
+	bool HTTPContacts::loadConfiguration(json_object *config) {
+	json_object *json_threshold;
+
+	HostCheck::loadConfiguration(config);
+
+	if (json_object_object_get_ex(config, "threshold", &json_threshold))
+		threshold = json_object_get_int64(json_threshold);
+	return (true);
+	}
+
+We need to tell to ntopng to instantiate the check class, to do so we need to modify :code:`src/HostChecksLoader.cpp`
+
+.. code:: C
+
+	void HostChecksLoader::registerChecks() {
+	HostCheck *fcb;
+
+	if ((fcb = new CountriesContacts()))   registerCheck(fcb);
+	[...]
+	if ((fcb = new HTTPContacts()))        registerCheck(fcb);
+	[...]
+	}
+
+These are the basic steps needed and must be replicated for all host, but even flow, to define a new host alert.
+What we can add now is a variable to be avaiable during the periodic update that store how many http flows an host have seen until that time.
+To do so we can modify the Host class adding a variable and a getter.  
+
+In :code:`/inlcude/Host.h` add the variable as well as a function to get it and ones to reset it.
+
+.. code:: C
+
+	class Host : public GenericHashEntry,
+				public Score,
+				public HostChecksStatus,
+			public HostAlertableEntity {
+	protected:
+	[...]
+	u_int32_t num_http_flows;
+	[...]
+	puiblic:
+	[...]
+	inline u_int32_t getNumHttpFlows() { return (num_http_flows); };
+	inline void resetNumHttpFlows() { num_http_flows = 0; };
+	}
+
+Now we need to update the variable every time a new http connection has been seen. To do so modify :code:`/src/Host.cpp`
+
+.. code:: C
+
+	void Host::initialize(Mac *_mac, int32_t _iface_idx,
+				u_int16_t _vlanId,
+						u_int16_t observation_point_id) {
+	if (_vlanId == (u_int16_t)-1) _vlanId = 0;
+	num_http_flows = 0;
+	[...]
+	}
+	[...]
+	void Host::incStats(u_int32_t when, u_int8_t l4_proto, u_int ndpi_proto,
+						ndpi_protocol_category_t ndpi_category,
+						custom_app_t custom_app, u_int64_t sent_packets,
+						u_int64_t sent_bytes, u_int64_t sent_goodput_bytes,
+						u_int64_t rcvd_packets, u_int64_t rcvd_bytes,
+						u_int64_t rcvd_goodput_bytes, bool peer_is_unicast) {
+	// http has the protocol id equal to 7
+	if(ndpi_proto == 7) num_http_flows++;
+	[...]
+	}
+
+Formatting the output
+----------------------
+
+One last thing we can do is to modify the locales in order to visualize both the check enable section and the alert launched in a readable format. 
```

Comment:
Please specify that the localized strings are those referenced in the alert or check definition